_posts/en/newsletters/2020-04-22-newsletter.md

```diff
@@ -84,7 +84,12 @@ release candidates.*
 [Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
 BOLTs][bolts repo].*
 
-- [Bitcoin Core #17595][] Merge #17595: guix: Enable building for `x86_64-w64-mingw32` target FIXME:dongcarl
+- [Bitcoin Core #17595][] adds support for [reproducibly building][topic
+  reproducible builds] Windows versions of Bitcoin Core using [GNU Guix][]
+  (pronounced "geeks"). Since its previous mention in the newsletter, this
+  effort to replace Gitian has gained [reproducibility across time][Bitcoin Core
```

Comment:
@dongcarl Even looking at the linked PR, I'm not sure what `guix time-machine` does or what "reproducibility across time" means.  Can you add a little more detail either in a comment or in the PR?

Reply:
Decided to simplify and remove the mention of that PR!